Sources/Logging/Logging.swift

```diff
@@ -21,6 +21,9 @@ public protocol LogHandler {
 
     // The log level
     var logLevel: Logger.Level { get set }
+
+    // A user-suppliable identifier
+    var label: String? { get set }
```

Comment:
could we make this `get` only? Then we can also remove all the locking around it. I think getting the name is good enough

Reply:
Fine by me!

Reply:
Too bad `let label: String` can't satisfy the optional protocol requirement.
Not sure if there is any benefit to:
```
let _label: String // Set in the initializer and used internally
var label: String? { return _label }
```